daemon/src/rollover_taker.rs

```diff
@@ -128,6 +132,7 @@ impl Actor {
             (self.oracle_pk, announcement),
             rollover_params,
             Role::Taker,
+            self.position,
```

Comment:
I'd suggest returning the position from `handle_rollover_accepted_taker` if you need it here.

Reply:
Thanks for the suggestion, that makes things a lot easier.